.github/workflows/docker_test.yml

```diff
@@ -27,15 +33,21 @@ jobs:
     name: Test docker images
     steps:
       - uses: actions/checkout@v4
+      - name: Install UV
+        run: |
+          pip install uv
+      - name: Sync dependencies
+        run: |
+          uv sync
```

Comment:
there is a setup uv action you can call. see https://github.com/langflow-ai/langflow/blob/nightly-fixes/.github/workflows/docker-build.yml#L104